diff --git a/build/Jamfile.v2 b/build/Jamfile.v2
index 8fd92e64..66b0d556 100644
--- a/libs/serialization/build/Jamfile.v2
+++ b/libs/serialization/build/Jamfile.v2
@@ -100,9 +100,9 @@ lib boost_serialization
     : 
     <toolset>msvc:<cxxflags>/Gy
     <toolset>msvc:<define>_SCL_SECURE_NO_WARNINGS
-    <toolset>clang:<cxxflags>"-fvisibility=hidden -fvisibility-inlines-hidden"
-    <toolset>gcc:<cxxflags>"-fvisibility=hidden -fvisibility-inlines-hidden"
-    <toolset>darwin:<cxxflags>"-fvisibility=hidden -fvisibility-inlines-hidden"
+#   <toolset>clang:<cxxflags>"-fvisibility=hidden -fvisibility-inlines-hidden"
+#   <toolset>gcc:<cxxflags>"-fvisibility=hidden -fvisibility-inlines-hidden"
+#   <toolset>darwin:<cxxflags>"-fvisibility=hidden -fvisibility-inlines-hidden"
     <toolset>gcc:<cxxflags>"-ftemplate-depth-255"
     <toolset>clang:<cxxflags>"-ftemplate-depth-255"
     <toolset>darwin:<cxxflags>"-ftemplate-depth-255"
@@ -114,9 +114,9 @@ lib boost_wserialization
     :     
     <toolset>msvc:<cxxflags>/Gy 
     <toolset>msvc:<define>_SCL_SECURE_NO_WARNINGS
-    <toolset>clang:<cxxflags>"-fvisibility=hidden -fvisibility-inlines-hidden"
-    <toolset>gcc:<cxxflags>"-fvisibility=hidden -fvisibility-inlines-hidden"
-    <toolset>darwin:<cxxflags>"-fvisibility=hidden -fvisibility-inlines-hidden"
+#   <toolset>clang:<cxxflags>"-fvisibility=hidden -fvisibility-inlines-hidden"
+#   <toolset>gcc:<cxxflags>"-fvisibility=hidden -fvisibility-inlines-hidden"
+#   <toolset>darwin:<cxxflags>"-fvisibility=hidden -fvisibility-inlines-hidden"
     <toolset>gcc:<cxxflags>"-ftemplate-depth-255"
     <toolset>clang:<cxxflags>"-ftemplate-depth-255"
     <toolset>darwin:<cxxflags>"-ftemplate-depth-255"
